### ISSUE TITLE
TypeError: this.port.on is not a function

### ISSUE THREAD

**Issue by thorikawa** [song940/node-bluetooth#6]
I tired this module. Just run an example on OSX Sierra (10.12.5) and got the following error while connecting to the device. Any help with this?

```
Found: 00-06-66-80-d2-94 with name RNBT-D294
2017-07-25 21:16:46.682 node[2148:28121] Update serviceMask RNBT-D294 0
finished
Found RFCOMM channel for serial port on RNBT-D294:  1
/Users/poly/ws/node-bluetooth-master/index.js:21
  this.port.on('data', function(buffer){
            ^

TypeError: this.port.on is not a function
    at new Connection (/Users/poly/ws/node-bluetooth-master/index.js:21:13)
    at /Users/poly/ws/node-bluetooth-master/index.js:80:31
```

**Comment by thorikawa** [comment]
Just in case, I use node v6.9.1 and node-bluetooth v1.1.2.

**Comment by thorikawa** [comment]
Seems to be fixed by PR #7, please review it.
